src/App.tsx

```diff
@@ -16,35 +22,96 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  const handleReverse = () => {
```

Comment:
[CODE STYLE] - The function name 'handleReverse' does not clearly describe what the function does. Consider renaming it to something more descriptive such as 'toggleReverse'.